### PR TITLE
added javascript/javascript1/week4 homework

### PR DESCRIPTION
**Pull request by pattnaikupasana**

### CODE REVIEW

File: javascript/javascript1/week4/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VoiceAssistant</title>
+    <script src="voiceAssistant.js"></script>
+</head>
+
+<body>
+
+</body>
+
```
Comment (dinythomas89) [inline javascript/javascript1/week4/index.html:15]: Normally we place <script> at the end of the body unless you want to do something with your js file before the structure or layout the done

File: javascript/javascript1/week4/voiceAssistant.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+
+let toDo=[];
+function getReply(command) {
+// Name actions
+{
```
Comment (dinythomas89) [inline javascript/javascript1/week4/voiceAssistant.js]: Why 2 open braces here?

**Review by dinythomas89 — APPROVED** [review]

**Review by dinythomas89 — APPROVED** [review]
Use Const or let in your js files. Use of var is deprecated after ES5

File: javascript/javascript1/week4/voiceAssistant.js (hunk 2 above)
Comment (pattnaikupasana) [reply javascript/javascript1/week4/voiceAssistant.js]: Sorry its wrongly written..I have corrected it.

**Comment by pattnaikupasana** [comment]
Thank you so much for the feedback 🙏

**Pull request merged**